tensorflow_addons/options.py

```diff
@@ -26,7 +25,7 @@
 If you want this warning to disappear, either make sure the TensorFlow installed
 is compatible with this version of Addons, or tell TensorFlow Addons to
 prefer using Python implementations and not custom C++/CUDA ones. You can do that
-by changing the TF_ADDONS_PY_OPS flag
+by changing the _TF_ADDONS_PY_OPS flag
```

Comment:
```suggestion
by changing the `TF_ADDONS_PY_OPS` flag
```
do not prefix it with underscore as this is telling how to set it with enviorment variable